src/ring_theory/power_series/well_known.lean

```diff
@@ -83,4 +84,72 @@ variables {A A'} (n : ℕ) (f : A →+* A')
 
 end field
 
+open ring_hom
+open finset nat
+
+variables {A : Type*} [comm_ring A]
+
+noncomputable def eval_mul_hom (a : A) : power_series A →+* power_series A :=
```

Comment:
This is a `def`, so the linter will fail in a few hours time because you didn't add a docstring!

Reply:
Also, this def belonsg in `power_series/basic`

Reply:
Ah ok. I did not know that, I have added it, thank you!